app/controllers/admin/userManagement.py

```diff
@@ -64,12 +66,27 @@ def removeProgramManagers():
         flash('Error while removing a manager.','warning')
         abort(500,"Error while trying to remove a manager.")
 
+@admin_bp.route('/admin/addNewProgramInfo', methods=['POST'])
+def addProgramInfo():
+    """Grabs info from JavaScript and then outputs it to logic function"""
+    programInfo = request.form #grabs user inputs from js
+    try:
+        return addNewProgramInfo(programInfo["senderName"],  #calls logic function to add data to database
+                                programInfo["replyToEmail"],
+                                programInfo["programId"])
```

Comment:
You should have a flash for success as well as failure, as there is no other UI indication that it succeeded.